src/Discord.Net.Rest/Entities/Channels/RestChannel.cs

```diff
@@ -46,7 +46,7 @@ internal static IRestPrivateChannel CreatePrivate(BaseDiscordClient discord, Mod
 
         //IChannel
         string IChannel.Name => null;
-        bool IChannel.IsNsfw => ChannelHelper.IsNsfw(this);
+        bool IChannel.IsNsfw => false;
```

Comment:
Why are we explicitly implementing IsNsfw? Users will just have to cast to IChannel to get access to this